### PR TITLE
refactor: refactor sass and add libraries

### PR DESCRIPTION
**Pull request by KaterinaKachann**

### CODE REVIEW

**Comment by KaterinaKachann** [comment]
[SASS in React not import (from)](https://trello.com/c/vqdaXP0Q/29-sass-in-react-not-import-from)

File: package.json [hunk 1]
```diff
@@ -1,95 +1,97 @@
...
+  "main": "webpack.base.config.js",
+  "scripts": {
+    "start": "react-scripts start",
+    "build": "react-scripts build",
+    "eject": "react-scripts eject",
+    "lint": "eslint --ext .ts",
+    "prettier-write": "prettier --write ./",
+    "prettier-check": "prettier --check ./",
+    "prepare": "husky install",
+    "node-scss": "sass --watch src/scss/style.scss public/style.css"
```
Comment (PoMaKoM) [inline package.json]: Убрать

File: package.json [hunk 2]
```diff
@@ -1,95 +1,97 @@
...
+    "@types/node": "^16.11.19",
+    "@types/react": "^17.0.38",
+    "@types/react-dom": "^17.0.11",
+    "nouislider": "^15.5.0",
+    "react": "^17.0.2",
+    "react-dom": "^17.0.2",
+    "react-scripts": "5.0.0",
+    "sass": "^1.46.0",
+    "sass-loader": "^12.4.0",
+    "scss-loader": "^0.0.1",
```
Comment (PoMaKoM) [inline package.json]: Убрать

File: public/index.html [hunk 3]
```diff
@@ -5,6 +5,7 @@
     <meta name="viewport" content="width=device-width, initial-scale=1" />
     <meta name="theme-color" content="#000000" />
     <meta name="description" content="Web site created using create-react-app" />
+    <link rel="stylesheet" href="./style.css" />
```
Comment (PoMaKoM) [inline public/index.html]: Убрать

**Review by gomunkool — DISMISSED** [review]

**Review by PoMaKoM — CHANGES_REQUESTED** [review]

**Review by PoMaKoM — APPROVED** [review]

**Pull request merged**